src/lib/src/ng-select.component.scss

```diff
@@ -1,275 +1,284 @@
+        @include box-sizing;
     }
-
-    > .as-control .as-arrow {
-      top: -2px;
-      border-color: transparent transparent #999;
-      border-width: 0 5px 5px;
-
-      &:hover {
-        border-color: transparent transparent #666;;
-      }
+    virtual-scroll {
+        display: block;
+        height: auto;
     }
-
-    .as-menu-outer{
-      visibility: visible;
+    &.opened {
+        >.ng-control {
+            border-bottom-right-radius: 0;
+            border-bottom-left-radius: 0;
+            background: #fff;
+            border-color: #b3b3b3 #ccc #d9d9d9;
+        }
+        >.ng-control .ng-arrow {
+            top: -2px;
+            border-color: transparent transparent #999;
+            border-width: 0 5px 5px;
+            &:hover {
+                border-color: transparent transparent #666;
+                ;
+            }
+        }
+        .ng-menu-outer {
+            visibility: visible;
+        }
     }
-  }
-
-  &.focused {
-    &:not(.opened) > .as-control {
-      border-color: #007eff;
-      box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 0 3px rgba(0, 126, 255, 0.1);
+    &.focused {
+        &:not(.opened)>.ng-control {
+            border-color: #007eff;
+            box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 0 3px rgba(0, 126, 255, 0.1);
+        }
     }
-  }
-
-  &.disabled > .as-control {
-    background-color: #f9f9f9;
-  }
-
-  .as-control {
-    background-color: #fff;
-    border-radius: 4px;
-    border: 1px solid #ccc;
-    color: #333;
-    cursor: default;
-    display: table;
-    border-spacing: 0;
-    border-collapse: separate;
-    height: 36px;
-    outline: none;
-    overflow: hidden;
-    position: relative;
-    width: 100%;
-
-    &:hover {
-      box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
+    &.disabled {
+        pointer-events: none;
+        >.ng-control {
+            background-color: #f9f9f9;
+        }
     }
-  }
-
-  .as-placeholder, &.as-single > .as-control .as-value {
-    bottom: 0;
-    color: #aaa;
-    left: 0;
-    line-height: 34px;
-    padding-left: 10px;
-    padding-right: 10px;
-    position: absolute;
-    right: 0;
-    top: 0;
-    max-width: 100%;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-
-    .as-value-label {
-      color:#333;
+    .ng-control {
+        background-color: #fff;
+        border-radius: 4px;
+        border: 1px solid #ccc;
+        color: #333;
+        cursor: default;
+        display: flex;
+        align-items: center;
+        border-spacing: 0;
+        border-collapse: separate;
+        min-height: 36px;
+        outline: none;
+        overflow: hidden;
+        position: relative;
+        width: 100%;
+        &:hover {
+            box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
+        }
+        .ng-value-container {
+            display: flex;
+            flex: 1;
+            align-items: center;
+            position: relative;
+            padding-left: 10px;
```

Comment:
In multiselect using `padding-left: 5px;` looks better because padding-top is also 5px.
![paddings](https://user-images.githubusercontent.com/3028012/31482422-f3202db4-af30-11e7-9c6c-84417d53553a.png)

Reply:
single has 10px, thats why there is no difference 
![image](https://user-images.githubusercontent.com/17824796/31501084-ebe4eb5c-af71-11e7-8a60-fc879123ab62.png)

Reply:
because its aligned with single ![image](https://user-images.githubusercontent.com/17824796/31501106-fb310bd6-af71-11e7-900f-34284d04da29.png)

Reply:
because its aligned with single 
![image](https://user-images.githubusercontent.com/17824796/31501148-199832e8-af72-11e7-934a-f2a0c2a5441e.png)

Reply:
I don't think you should align with single. In single select it's ok because top and bottom paddings are also looks the same because it's a text. In multiselect adding this extra left padding make look it ugly.
![ugly](https://user-images.githubusercontent.com/3028012/31535591-4e6b9bee-b004-11e7-8441-2c4640ca8a22.png)
![not-ugly](https://user-images.githubusercontent.com/3028012/31535606-554b17f0-b004-11e7-8a63-3509cac488d9.png)

I added `padding-left: 7px` because 2px is for borders. Just to be sure that placeholders are aligned in the same line then no items are selected.

Reply:
Slack multiselect example:
![slack](https://user-images.githubusercontent.com/3028012/31537145-c18aa890-b009-11e7-8486-b122e353bf8f.png)